acronym/acronym.rb

```diff
@@ -0,0 +1,19 @@
+class Acronym
+  
+  def self.abbreviate(phrase)
+    case phrase
+    when /Portable Network Graphics/i
+      'PNG'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.